src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/infrastructure/config/SecurityConfiguration.java

```diff
@@ -3,41 +3,53 @@
 import static org.springframework.security.config.http.SessionCreationPolicy.STATELESS;
 
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.List;
+import javax.servlet.http.HttpServletResponse;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
 import org.springframework.security.authentication.AuthenticationManager;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationConverter;
 import uk.gov.hmcts.reform.auth.checker.core.RequestAuthorizer;
 import uk.gov.hmcts.reform.auth.checker.core.service.Service;
 import uk.gov.hmcts.reform.auth.checker.core.user.User;
 import uk.gov.hmcts.reform.auth.checker.spring.serviceanduser.AuthCheckerServiceAndUserFilter;
+import uk.gov.hmcts.reform.auth.checker.spring.serviceonly.AuthCheckerServiceOnlyFilter;
+import uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.security.AuthorizedRolesProvider;
+import uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.security.SpringAuthorizedRolesProvider;
 
 @Configuration
 @ConfigurationProperties(prefix = "security")
 @EnableWebSecurity
 public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
 
     private final List<String> anonymousPaths = new ArrayList<>();
+
     private final RequestAuthorizer<User> userRequestAuthorizer;
     private final RequestAuthorizer<Service> serviceRequestAuthorizer;
     private final AuthenticationManager authenticationManager;
+    private final Converter<Jwt, Collection<GrantedAuthority>> authoritiesConverter;
+
+    private Boolean useOpenId = Boolean.FALSE;
```

Comment:
Maybe I've missed it, but should this be coming from config here?

Reply:
yes, default is in `application.yaml` I setup this default just in case, but it seems it is a bit confusing, let me remove it and leave only default in yaml file

Reply:
👍